apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -83,9 +84,15 @@ type WorkspaceConfig struct {
 	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
 	// +kubebuilder:validation:MaxLength=63
 	PVCName string `json:"pvcName,omitempty"`
-	// StorageClassName defines and optional storageClass to use for persistent
+	// StorageClassName defines an optional storageClass to use for persistent
 	// volume claims created to support DevWorkspaces
 	StorageClassName *string `json:"storageClassName,omitempty"`
+	// StorageClassSize defines an optional map of sizes of persistent volume claims for each storage
+	// class used by DevWorkspaces.
+	// The available keys to the map are: "common", "per-workspace".
```

Comment:
Maybe worth noting that `common` is used for the `async` storageClass as well.

Reply:
Done :+1:  Note that the documentation maybe could be better, not sure about it yet. Also, it's subject to change once I find proper default sizes for PVC's